README.md

```diff
@@ -1,5 +1,28 @@
-# loopback-connector-cos
+<h2>Instructions on how to use this loopback connector with COS + APIC </h2>
 
-IBM Cloud Object Storage connector for LoopBack
+<p><i>Prerequisites: </i> 
+	<ul>
+			<li> 1) an APIC app set up</li>
+			<li> 2) an instance of Cloud Object Storage created</li>
+			<li> 3) service credentials generated for your COS instance </li>
+			<li> 4) an instance of Key Protect created </li>
+			<li> 5) a Key Protect root key created inside</li>
+			<li> 6) A bucket created inside your instance of COS </li>
+	</ul>
 
-_TODO: Add more content per https://loopback.io/doc/en/contrib/readmes.html_
```

Comment:
It would be great if you could follow our guidelines for README files: https://loopback.io/doc/en/contrib/readmes.html

Not a big deal though, this pull request can be landed with the currently-proposed content too.